src/components/Board.tsx

```diff
@@ -33,6 +33,9 @@ const useStyles = makeStyles({
 });
 
 const Template: React.FC = () => {
+  React.useEffect(() => {
```

Comment:
Hrmmmmmm, I feel like there's no way we need this on every page

also, why is this component named Template. 

Neither have to do with this PR specifically, so non-blocking.

Reply:
I also noticed that, asked Kimin to rename it